packages/php-wasm/fs-journal/src/lib/fs-journal.ts

```diff
@@ -178,11 +178,11 @@ export function journalFSEvents(
 		php[__private__dont__use].journal.unbind();
 		delete php[__private__dont__use].journal;
 	}
-	php.addEventListener('runtime.beforedestroy', unbindFromOldRuntime);
+	php.addEventListener('runtime.beforeExit', unbindFromOldRuntime);
```

Comment:
Technically the PHP instance can still be reinitialized with another PHP runtime so it's not destroyed